website/blog/2022/01/21/apisix-xrpc-details-and-miltilingual.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "xRPC Will Be Released, Get More Details About APISIX Ecosystem"
+authors:
+  - name: "Jinchao Shuai"
+    title: "Author"
+    url: "https://github.com/shuaijinchao"
+    image_url: "https://avatars.githubusercontent.com/u/8529452?v=4"
+  - name: "Sylvia"
+    title: "Technical Writer"
+    url: "https://github.com/SylviaBABY"
+    image_url: "https://avatars.githubusercontent.com/u/39793568?v=4"
+keywords: 
+- Multiple protocols
+- Apache APISIX
+- Java
+- Multilingual
+- Ecology
+description: This article brings you Apache APISIX's upcoming xRPC framework and related details, as well as a detailed presentation of Apache APISIX in multi-language development support.
+tags: [Ecosystem]
```

Comment:
we decide to support multiple tags after discussion, see https://github.com/apache/apisix-website/issues/744 
```suggestion
tags: [Technology,Ecosystem]
```